platforms/macos/src/event.rs

```diff
@@ -5,14 +5,14 @@
 
 use accesskit::{Live, NodeId};
 use accesskit_consumer::{DetachedNode, FilterResult, Node, TreeChangeHandler, TreeState};
-use objc2::{
-    foundation::{NSInteger, NSMutableDictionary, NSNumber, NSObject, NSString},
-    msg_send, Message,
+use icrate::{
+    AppKit::*,
+    Foundation::{NSInteger, NSMutableDictionary, NSNumber, NSString},
 };
+use objc2::{msg_send, runtime::Object, Message};
 use std::rc::Rc;
 
 use crate::{
-    appkit::*,
     context::Context,
     node::{filter, filter_detached, NodeWrapper},
 };
```

Comment:
Nit:

I'd recommend using [`NSAccessibilityNotificationName`](https://docs.rs/icrate/0.0.1/icrate/AppKit/type.NSAccessibilityNotificationName.html) in `QueuedEvent::Generic::notification` below, since that is actually the correct type.

Likewise for `QueuedEvent::Announcement::priority` being [`NSAccessibilityPriorityLevel`](https://docs.rs/icrate/0.0.1/icrate/AppKit/type.NSAccessibilityPriorityLevel.html)